indigo_app/templates/indigo_app/tasks/my_tasks.html

```diff
@@ -0,0 +1,94 @@
+{% extends "main.html" %}
```

Comment:
Pls add a page title (override the `title` block)

Reply:
Updated to `My tasks`